src/zepto.js

```diff
@@ -678,6 +678,8 @@ var Zepto = (function() {
         $this.css(props)
       })
       if (!this.length) return null
+      if (!Zepto.contains(document.documentElement, this[0]))
```

Comment:
fwiw, `try…catch` is really expensive. perhaps we should feature detect if the if is necessary (the first time the function is called), store the result and exit early from the if when it isn't.